docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: '3.8'
+networks:
+  db-data-net:
+    driver: bridge
+
+services:
+  pythonapp:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - "8081:8080"
+    environment:
+      ENV: local
+    networks:
+      - db-data-net
+    depends_on:
+      - mysql
+    restart: unless-stopped
+
+  mysql:
+    build:
+      context: .
+      dockerfile: Dockerfile.mysql
```

Comment:
Both services (`pythonapp` and `mysql`) are using the same build context ('.'). Ensure that this is intentional, as they might require different contexts or Dockerfiles.